pkg/runtime/runtime.go

```diff
@@ -157,6 +158,10 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	if err != nil {
 		log.Warnf("failed to load components: %s", err)
 	}
+	err = a.beginComponentsUpdates()
+	if err != nil {
+		log.Warnf("failed to watch component updates: %s", err)
```

Comment:
Should it be error (may be fatal)? Earlier the updates were pushed by operator to daprd, so if one update push failed future updates will continue. In this new model of daprd streaming from operator, if it fails to setup streaming, it will not get future updates.
I am not sure of correct behavior here, adding this comment to discuss further.

Reply:
That's true, as commented below, we are not 100% clear yet on what we should do with component updates. The situation today is that Dapr is getting notified but doing nothing with the updates.

In order to keep this PR focused on transitioning from HTTP to gRPC, I would save this for a later discussion and PR.